source/MaterialXRender/Handlers/OiioImageLoader.cpp

```diff
@@ -53,31 +101,57 @@ bool OiioImageLoader::acquireImage(const FilePath& filePath,
     }
 
     OIIO::ImageSpec imageSpec = imageInput->spec();
-    if (imageSpec.format == OIIO::TypeDesc::HALF)
+    switch (imageSpec.format.basetype)
     {
-        // Due to display issue with 16-bit tiled exrs,
-        // treat as 32-bit float.
-        imageSpec.set_format(OIIO::TypeDesc::FLOAT);
-    }
+        case OIIO::TypeDesc::UINT8:
+        {
+            imageDesc.baseType = ImageDesc::BASETYPE_UINT8;
```

Comment:
For completeness should restrictions be checked here and for FLOAT below as well?